wiki-page-creator.html

```diff
@@ -39,7 +39,7 @@
   var formElements = ['sprint', 'milestone', 'bug', 'points'];
   var elements = ['input', 'output', 'url', 'updateButton'].concat(formElements);
   var reTemplate = /<([^>]+)>/g;
-  var reRemoveHead = /^[\s\S]*?\n(?===)/; // \s\S catches also the newline character
+  var reRemoveHead = /^[\s\S]*?(?:\n|^)(?==)/; // \s\S catches also the newline character
```

Comment:
now we only need one `=` (so your `=Bugs=` works) and this can either be a new line or the start of the string before that `=`. I checked with your template and it works fine with yours (while still working for mine :) )